js-classes-inheritance/script.js

```diff
@@ -0,0 +1,94 @@
+class Builder {
+    constructor (arg) {
+        this.arg = arg;
```

Comment:
arg is not so speaking name for variable like "accumulator" or "sum".
I'd suggest to rename)